src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Represents the user's list of tasks.
+ * Contains basic operations to manipulate tasks in the list.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public ArrayList<Task> getTasks() {
+        return this.tasks;
+    }
+
+    public Task remove(int index) {
+        return tasks.remove(index);
+    }
+
+    public void add(Task t) {
+        tasks.add(t);
+    }
+
+    public int size() {
```

Comment:
Maybe shall we change the naming of this method to ```getSize()```? As it may sound more like an action.